ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/button-behavior/button-behavior.tsx

```diff
@@ -28,7 +29,7 @@ type Props = {
   style?: React.CSSProperties
 }
 
-const Root = styled<{}, 'div'>('div')`
+const Root = styled.div`
```

Comment:
❓ Is there an issue with reducing this form? I thought that @andrewkfiedler mentioned that there may be at a UI roadmap/training session. It sounds like this is the doing of the formatter, though.

Reply:
This form is fine.  It only needs to be the long form if we're passing in props in addition to the theme.